uvicorn/main.py

```diff
@@ -391,6 +392,8 @@ def run(app: typing.Union[ASGIApplication, str], **kwargs: typing.Any) -> None:
         Multiprocess(config, target=server.run, sockets=[sock]).run()
     else:
         server.run()
+        if config.uds:
+            os.remove(config.uds)
```

Comment:
Maybe I'm mistaken, but can't we have a single one of those outside the conditional?

```suggestion
    if config.uds:
        os.remove(config.uds)
```

Reply:
I guess we can, but it's not right, because we bind inside them. :thinking: Right?

Reply:
it looks simpler yes, I removed occurrences in the basereload and multiprocess shutdown method and put it outside